src/i18n/en.json

```diff
@@ -501,5 +501,26 @@
     },
     "TITLE": "Sign up"
   },
+  "GROUP": {
+    "CREATE_GROUP_TITLE": "Create Group",
+    "GROUP_NAME": "Group Name",
+    "GROUP_DESCRIPTION": "Description",
+    "CONTINUE": "Continue",
+    "SAVED": "Group Saved",
+    "SAVED_MESSAGE": "We were unable to create your Group at the moment, but if you are offline don't worry it's saved and will be uploaded once connectivity is back",
+    "CREATING": "We're creating your group",
+    "SUCCESS": "Created successfully",
+    "GROUP_ICON": "Icon",
+    "SPECIES_GROUP_COVERAGE": "Species Group Coverage",
+    "HABITATS_COVERED": "Habitats Covered",
+    "SPATIAL_COVERGE": "Spatial Coverage",
+    "TAGS": "Tags",
+    "GROUP_ADMIN_TITLE": "Administrators",
+    "GROUP_INVITE_FOUNDER": "Invite Founders",
+    "GROUP_INVITE_MODERATORS": "Invite Moderators",
```

Comment:
Please name keys properly for example there's no need to prefix keys with group if they already are under group